canonical/store-components
# Add topic card

## Done
Added a topic card

## QA
- Go to https://store-components-53.demos.haus/?path=/story/topiccard--default-story
- Check that there is a topic card
